native/src/main/java/io/ballerina/stdlib/websocket/ReturnStreamUnitCallBack.java

```diff
@@ -72,8 +72,7 @@ public void notifySuccess(Object response) {
                 sendTextMessageStream(StringUtils.fromString(content), promiseCombiner);
                 Thread.startVirtualThread(() -> {
                     try {
-                        Object result = runtime.startIsolatedWorker(bObject, STREAMING_NEXT_FUNCTION, null,
-                                null, null).get();
+                        Object result = runtime.callMethod(bObject, STREAMING_NEXT_FUNCTION, null);
```

Comment:
We should pass strandmetadata with concurrent safe true

Reply:
Check in other places too

Reply:
Will fix them

Reply:
Fixed in ad61731008116e64e97785cca3b3c54cbd6b21b0